src/uct/base/uct_iface.h

```diff
@@ -388,7 +388,7 @@ typedef struct uct_iface_mpool_config {
  * TL Memory pool object initialization callback.
  */
 typedef void (*uct_iface_mpool_init_obj_cb_t)(uct_iface_h iface, void *obj,
-                uct_mem_h memh);
+              uct_mem_h memh);
```

Comment:
looks like an alignment is still odd, it should be:
```
typedef void (*uct_iface_mpool_init_obj_cb_t)(uct_iface_h iface, void *obj,
                                              uct_mem_h memh);
```

Reply:
yep, fixed... this is not final changes - looking at CPP check false positives for now